packages-exp/auth-exp/src/core/strategies/email.ts

```diff
@@ -65,3 +65,28 @@ export async function sendEmailVerification(
     await user.reload();
   }
 }
+
+export async function verifyBeforeUpdateEmail(
```

Comment:
This should be reexported in `index.ts` right?

Reply:
good catch! thanks